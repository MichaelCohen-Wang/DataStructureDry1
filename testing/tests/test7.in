add_ship 510 21
add_ship 348 29
treason 510 348
add_pirate 423 348 2
remove_pirate 423
remove_ship 510
get_treasure 137
update_pirate_treasure 471 -1