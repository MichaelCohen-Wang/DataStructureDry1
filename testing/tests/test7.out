add_ship: SUCCESS
add_ship: SUCCESS
treason: FAILURE
add_pirate: SUCCESS
remove_pirate: SUCCESS
remove_ship: SUCCESS
get_treasure: FAILURE
update_pirate_treasure: FAILURE
